workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityComponentDataGenerator.tt

```diff
@@ -94,9 +121,7 @@ namespace <#= qualifiedNamespace #>
 <# if (!fieldDetails.IsBlittable) { #>
                 component.<#= fieldDetails.CamelCaseName#>Handle = <#= qualifiedNamespace #>.<#= componentDetails.ComponentName #>.ReferenceTypeProviders.<#= fieldDetails.PascalCaseName #>Provider.Allocate(world);
 <# } #>
-                {
```

Comment:
Removal of the `{}` around these may have harmful effects (I remember some deserialization strings used local variables which may clash if there are multiple properties of the same type in the same component) - a fix would be to place the `{}` into those string generators.

Reply:
Rolling this back, just to be safe.